src/elexclarity/formatters/const.py

```diff
@@ -8,6 +8,13 @@
         'US Senate (Loeffler) - Special/Senado de los EE.UU. (Loeffler) - Especial': 'S2',
         'US Senate (Perdue)': 'S',
         'US Senate (Perdue)/Senado de los EE.UU. (Perdue)': 'S',
+        # 2022 primaries
+        'US Senate - Rep': 'S',
+        'US Senate - Dem': 'S',
+        'Governor - Rep': 'G',
+        'Governor - Dem': 'G',
+        'Secretary of State - Rep': 'R',
+        'Secretary of State - Dem': 'R'
```

Comment:
this returns 159 counties for each?

Reply:
this returns 147 counties, because 12 of the counties 404 on the clarity site 

(Bacon, Crawford, Fannin, Fulton, Gwinnett, Heard, Macon, Pierce, Schley, Towns, Washington, Wheeler)

when I use Chris' fix here https://github.com/washingtonpost/elex-clarity/pull/19, it returns 158 counties